src/view/screens/Search/Search.tsx

```diff
@@ -172,6 +227,7 @@ function SearchScreenSuggestedFollows() {
       contentContainerStyle={{paddingBottom: 1200}}
       keyboardShouldPersistTaps="handled"
       keyboardDismissMode="on-drag"
```

Comment:
Also if we add `onEndReachedThreshold={2}` (I think 3 might be a bit too soon since there are not that many items per page, unless we want to add more) we shouldn't hit the bottom as much.